turkce/catalla-kaz/kus/kus.md

```diff
@@ -0,0 +1,13 @@
+Kusmaya başladın.
+
+O ses kulaklarından hiç gitmediği için sürekli kusmaya devam ediyordun.
+
+Biraz duvarları yesem geçer diye düşündün.
+
+Ama ya geçmezse?
+
+Bir yol bulup ilerlemek de bir seçenekti.
+
+Ne yapacaksın?
+
+[Duvarları yeme başla?](../turkce/duvarlari-ye/duvarlari-ye.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.